CMakeLists.txt

```diff
@@ -32,9 +32,6 @@ cmake_minimum_required (VERSION 2.6)
 project(muser)
 include(GNUInstallDirs)
 
-
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wextra")
```

Comment:
Why to remove this? We do want to catch the compiler warnings all over the code.

Reply:
Yeah, this was a mistake; I've reworked how we set the flags in both places.